Doc/extending/extending.rst

```diff
@@ -253,6 +268,11 @@ needed to ensure that it will not be discarded, causing :c:data:`!SpamError` to
 become a dangling pointer. Should it become a dangling pointer, C code which
 raises the exception could cause a core dump or other unintended side effects.
 
+Also note that there is no :c:func:`Py_DECREF` call to remove this reference.
```

Comment:
Unlike the previous two notes above, this paragraph mentions a kind of workaround.  So, I'd prefer "missing" to "no", or something like "for now"?